installing/installing_with_agent_based_installer/understanding-disconnected-installation-mirroring.adoc

```diff
@@ -0,0 +1,33 @@
+:_content-type: ASSEMBLY
+[id="understanding-disconnected-installation-mirroring"]
+= Understanding disconnected installation mirroring
+include::_attributes/common-attributes.adoc[]
+:context: understanding-disconnected-installation-mirroring
+
+toc::[]
+// Reusing applicable content from Disconnected installation mirroring assembly
+
+You can use a mirror registry to ensure that your clusters only use container images that satisfy your organizational controls on external content. Before you install a cluster on infrastructure that you provision in a disconnected environment, you must mirror the required container images into that environment. To mirror container images, you must have a registry for mirroring.
```

Comment:
I wonder if the statement "ensure that your clusters only use container images that satisfy your organizational controls on external content" is correct, as the container images started out external, they were just mirrored and stored locally. Does that mean they are no longer "external" content?